benefits/core/models/transit.py

```diff
@@ -182,6 +187,20 @@ def eligibility_api_public_key_data(self):
         """This Agency's public key as a string."""
         return self.eligibility_api_public_key.data
 
+    @property
+    def littlepay_config(self):
+        if hasattr(self, "transitprocessorconfig") and hasattr(self.transitprocessorconfig, "littlepayconfig"):
+            return self.transitprocessorconfig.littlepayconfig
+        else:
+            return None
+
+    @property
+    def switchio_config(self):
```

Comment:
@machikoyasuda related to your question about "How to decide if the agency is using Littlepay or Switchio" e.g. for the help page...

This PR adds a `littlepay_config` and `switchio_config` prop to the `TransitAgency` model. One of them should have a value for active agencies.